packages/contracts/package.json

```diff
@@ -32,6 +32,9 @@
     "test:contracts": "hardhat test --show-stack-traces",
     "test:gas": "hardhat test \"test/contracts/OVM/execution/OVM_StateManager.gas-spec.ts\" --no-compile --show-stack-traces",
     "test:coverage": "NODE_OPTIONS=--max_old_space_size=8192 hardhat coverage",
+    "test:slither": "slither .",
+    "pretest:slither": "rm -f @openzeppelin && rm -f @ens && rm -f hardhat && ln -s ../../node_modules/@openzeppelin @openzeppelin && ln -s ../../node_modules/@ens @ens && ln -s ../../node_modules/hardhat hardhat",
+    "posttest:slither": "rm -f @openzeppelin && rm -f @ens && rm -f hardhat",
```

Comment:
Should we have a command that combines all of these steps into one?

Reply:
These "pre" and "post" hooks are an npm feature and execute automatically before and after the command. There's nothing more we need to do. I've had to link these packages that way as slither does not work well in a monorepo setup and fails to read node modules from the root.

Reply:
Oh cool! I didn't realize the hooks would get picked up even with the `:` in them.